packages/frontend/src/main/components/onboarding/GettingStartedSteps.vue

```diff
@@ -14,242 +14,162 @@
 
       <v-window v-model="onboarding" class="pb-3">
         <v-window-item>
-          <v-card class="transparent elevation-0 text-center" color="transparent">
-            <v-img
-              class="white--text align-end mb-3"
-              height="200px"
-              src="@/assets/onboardingsplash.png"
-            ></v-img>
-            <v-card-title class="display-1 justify-center mb-5">
-              👋 Welcome to Speckle!
-            </v-card-title>
-            <v-card-subtitle class="subtitle-1 justify-center mb-5">
-              The Open Data Infrastructure for the Built Environment.
-            </v-card-subtitle>
-            <v-card-text class="body-1 text--primary">
-              <p>
-                Engineers, designers, hackers and entire organizations rely on us for
-                interoperability, automation and collaboration to deliver better buildings, faster.
-                <br />
-                Please select your role:
-              </p>
-            </v-card-text>
-            <v-container fluid>
-              <v-row dense>
-                <v-col cols="5" offset="1">
-                  <v-hover>
-                    <v-card
-                      slot-scope="{ hover }"
-                      height="100%"
-                      :class="`pa-5 elevation-${hover ? 10 : 10}`"
-                      @click="nextUser"
-                    >
-                      <h1>👷‍♀️</h1>
-                      <v-card-title class="justify-center">Designer or Manager</v-card-title>
-                      <v-card-subtitle class="justify-center">
-                        Architect, engineer, BIM Manager, etc.
-                      </v-card-subtitle>
-                    </v-card>
-                  </v-hover>
-                </v-col>
-                <v-col cols="5" offset="0">
-                  <v-hover>
-                    <v-card
-                      slot-scope="{ hover }"
-                      height="100%"
-                      :class="`pa-5 elevation-${hover ? 10 : 10}`"
-                      @click="nextDev"
-                    >
-                      <h1>👩‍💻</h1>
-                      <v-card-title class="justify-center">Developer</v-card-title>
-                      <v-card-subtitle class="justify-center">
-                        Software engineer, computational designer, hacker
-                      </v-card-subtitle>
-                    </v-card>
-                  </v-hover>
-                </v-col>
-              </v-row>
-            </v-container>
-            <p class="my-7"><strong>Let us give you a quick tour! 🙋‍♂️</strong></p>
+          <v-card class="transparent elevation-0 text-center" color="transparent" align="start">
+            <v-img class="align-start mb-3" src="@/assets/onboarding-1.png" :aspect-ratio="16 / 9">
```

Comment:
Would be nice to optimize these images further:
1. Compress as much as possible (maybe this is already done)
2. Generate a .webp version of these images and fallback to PNG if browsers don't support them https://usefulangle.com/post/114/webp-image-in-html-with-fallback

(In the future we should make this process automatic so that webpack handles it on build)